usersession/userd/launcher_test.go

```diff
@@ -74,7 +74,7 @@ func (s *launcherSuite) TestOpenURLWithNotAllowedScheme(c *C) {
 }
 
 func (s *launcherSuite) TestOpenURLWithAllowedSchemeHappy(c *C) {
-	for _, schema := range []string{"http", "https", "mailto", "snap", "help", "apt", "zoommtg", "zoomus", "zoomphonecall", "slack", "msteams"} {
+	for _, schema := range []string{"com.cloudflare.warp", "http", "https", "mailto", "snap", "help", "apt", "zoommtg", "zoomus", "zoomphonecall", "slack", "msteams"} {
```

Comment:
can you add this at the end of the list?

Reply:
For sure! I thought it was alphabetical but it looks like its not. Done!